### ISSUE TITLE
Missing ARIA attributes

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1108]
Success Criteria 4.1 in WCAG 2 (A - must) fails due to missing ARIA attributes.

Failing elements on front page:
```HTML
<a href="#" role="button" class="django-cradmin-menu-mobiletoggle" ng-click="cradminMenuTogglePressed()" ng-class="{'django-cradmin-menu-mobile-toggle-button-expanded': cradminMenuDisplay}" aria-pressed="">
```

**Comment by Eskalol** [comment]
Fix is in cradmin_legacy and will be released with `milestone-5.1`-branch. The problem was wrong usage of aria-pressed values.
